Tiltfile

```diff
@@ -410,7 +410,7 @@ include_user_tilt_files()
 load("ext://cert_manager", "deploy_cert_manager")
 
 if settings.get("deploy_cert_manager"):
-    deploy_cert_manager()
+    deploy_cert_manager(version = settings.get("cert_manager_version"))
```

Comment:
It looks like `cert_manager_version` was unused, and we were getting the default version of v1.3.1 here.